samples/Cancellation/HttpTriggerWithCancellation.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Cancellation
+{
+    public class HttpTriggerWithCancellation
+    {
+        private readonly ILogger _logger;
+
+        public HttpTriggerWithCancellation(ILoggerFactory loggerFactory)
```

Comment:
Should we consider adding this function to the existing FunctionApp sample instead of new project ?

Reply:
This sample is already in the FunctionApp project but could use an update

Reply:
Yup, this sample already exists in the FunctionApp sample, and I think it's best to keep it there. Creating this as a draft in case @cloudmelon needs the cancellation sample to be it's own project

Reply:
I need a project sample with functions that we can just use 'func start' to test out the capabilities

Reply:
> I need a project sample with functions that we can just use 'func start' to test out the capabilities, can we sign-off this one ?

FWIWI, We should be able to do that in the [FunctionApp](https://github.com/Azure/azure-functions-dotnet-worker/tree/main/samples/FunctionApp) sample.